index.js

```diff
@@ -21,15 +21,10 @@ fontmachine.makeGlyphs = function(opts, callback) {
     var stack = [];
     fontnik.load(opts.font, function(err, faces) {
         if (err) return callback(err);
-        // temp workaround: fill this out
+
```

Comment:
@mikemorris @jakepruitt in your font exploration did you manage to find any font files that contain >1 face?